build.js

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env node
 
-import child_process from 'child_process';
-import fs from 'fs';
-import os from 'os';
-import path from 'path';
-import process from 'process';
+import child_process from 'node:child_process';
+import fs from 'node:fs';
+import os from 'node:os';
+import path from 'node:path';
+import process from 'node:process';
```

Comment:
Is this a standard anywhere? Or something deno invented? Please link the upstream documentation at least. So is this supported for all our supported nodejs versions? 18 and up?

https://nodejs.org/api/esm.html#node-imports

What is the benefit of these imports however?

Reply:
We already did *some* imports with `node:` prefixes before, just not consistently. E.g. here:

> pkg/lib/esbuild-compress-plugin.js:import child_process from 'node:child_process';

That's how they are [documented](https://nodejs.org/api/child_process.html). Your link says the same.